histolab/masks.py

```diff
@@ -132,10 +133,22 @@ def _regions(regions: List[Region], n: int = 1) -> List[Region]:
 
 
 class TissueMask(BinaryMask):
-    """Object that represent the whole tissue area mask.
+    """Object that represent the whole tissue area mask."""
 
-    The tissue within the slide or tile is automatically detected through a predefined
-    chain of filters."""
+    def __init__(self, *filters: ImageFilter) -> None:
+        """
+        Create a new tissue mask.
+
+        If custom image filters are specified, those are used instead the default ones.
+        By default, the tissue within the slide or tile is automatically detected
+        through a predefined chain of filters.
+
+        Parameters
+        ----------
+        filters : ImageFilter
```

Comment:
```suggestion
        *filters : ImageFilter
```
Otherwise I wouldn't know how to specify that that's an arbitrary number of parameters